bot/settings.py

```diff
@@ -12,6 +12,9 @@
 # event api key
 EVENT_API_KEY = os.environ.get("EVENT_API_KEY")
 
+# wolfram appid
+APPID = os.environ.get("WOLFRAM_APPID")
```

Comment:
`appid` needs to be descriptive as to what it is. It should match the env var, so it should be `WOLFRAM_APPID`